src/dbg/analysis/recursiveanalysis.cpp

```diff
@@ -51,7 +51,7 @@ void RecursiveAnalysis::SetMarkers()
         {
             FunctionDelRange(start, end, false /* Do not override user-defined functions */);
             LoopDeleteRange(start, end); // clear loop range in function
-            XrefDelRange(start, end); // clear xrefs in function
+            //XrefDelRange(start, end); // clear xrefs in function

```

Comment:
Maybe it's better to remove only xrefs for which both the source and destination are within the range. The real problem is that xref with source outside the range is removed.